ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -362,6 +392,31 @@ impl Mutation {
         ))
     }
 
+    async fn create_repository(ctx: &Context, name: String, git_url: String) -> Result<ID> {
+        Ok(ctx
+            .locator
+            .repository()
+            .create_repository(name, git_url)
+            .await?)
+    }
+
+    async fn delete_repository(ctx: &Context, id: ID) -> Result<bool> {
+        Ok(ctx.locator.repository().delete_repository(id).await?)
+    }
+
+    async fn update_repository(
+        ctx: &Context,
+        id: ID,
+        name: String,
+        git_url: String,
+    ) -> Result<bool> {
+        Ok(ctx
+            .locator
+            .repository()
+            .update_repository(id, name, git_url)
+            .await?)
+    }
+
```

Comment:
Let's don't allow updating a repository (so front end user have to delete an repository and create a new one).